Controllers/AccountController.cs

```diff
@@ -17,7 +17,7 @@ public IActionResult SignIn()
         {
             var redirectUrl = Url.Action(nameof(HomeController.Index), "Home");
             return Challenge(
-                new AuthenticationProperties { RedirectUri = redirectUrl },
+                new AuthenticationProperties { RedirectUri = redirectUrl, IsPersistent=true },
```

Comment:
Should change this to "IsPersistent = true" 
(spaces on both sides of the "=" sign)

Reply:
Made this change.